delivery_dhl_parcel/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Studio73 - Ethan Hildick <ethan@studio73.es>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Delivery DHL Parcel",
+    "summary": "Delivery Carrier implementation for DHL Parcel using their API",
+    "version": "14.0.1.0.0",
+    "category": "Stock",
+    "website": "https://github.com/OCA/l10n-spain",
+    "author": "Studio73, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
```

Comment:
¿Tenéis claro que sea LGPL la licencia adecuada?

Reply:
Esto es por copia y pega, lo modificaré a AGPL al modificar el resto de los comentarios